src/angularjs/src/styles/components/_map.scss

```diff
@@ -27,3 +27,17 @@
     color: #000;
   }
 }
+
+.pfb-control-legend {
+  padding: 5px;
+  line-height: 18px;
+  color: #555;
+
+  i {
+    width: 18px;
+    height: 18px;
+    float: left;
+    margin-right: 8px;
+    opacity: 0.7;
+  }
+}
```

Comment:
Add the following styling and print button issue will be fixed. Other than that I'm happy 👍 
```
.pfb-control-map-button.leaflet-control {
  position: absolute;
  top: 60px;
  right: 0px;
  z-index: 1;
}
```